esp/services/ws_dfu/ws_dfuXRefService.cpp

```diff
@@ -555,12 +559,15 @@ bool CWsDfuXRefEx::onDFUXRefList(IEspContext &context, IEspDFUXRefListRequest &r
     {
         context.ensureFeatureAccess(FEATURE_URL, SecAccess_Read, ECLWATCH_DFU_XREF_ACCESS_DENIED, "WsDfuXRef::DFUXRefList: Permission denied.");
 
+        Owned<IPropertyTree> xrefNodeTree = createPTree("XRefNodes");
+
+#ifndef _CONTAINERIZED
+        //In bare metal environment, each physical cluster is an XRef node.
```

Comment:
I'm not that familiar with how this service interacts with XREF.  This change correctly removes the process clusters (which should match DFU groups) from the list.  

@ghalliday @jakesmith 
But should the storage planes be on the list instead?  Or is that not necessary in the cloud?